Swerve2022/src/main/java/frc/robot/Constants.java

```diff
@@ -51,9 +53,170 @@
     public static final int    kLRCANCoderCANID       = 3;
     public static final int    kRRCANCoderCANID       = 4;
 
+    // Drivetrain specs
     public static final double kWheelDiaMeters        = Units.inchesToMeters(4.0); // 4in (39.37 in/meter)
     public static final double kGearRatio             = 6.75;
     public static final double kEncoderMetersPerCount = (kWheelDiaMeters * Math.PI) / (Falcon500.kEncoderCPR) / kGearRatio;
+    public static final double kTrackWidthMeters      = 0.6477;        // Measured track width
+
+    // Joystick tuning
+    public static final double kDriveXScaling         = 1.0;           // 1.0 is no scaling
+    public static final double kDriveYScaling         = 1.0;           // 1.0 is no scaling
+    public static final double kQuickTurnScaling      = 0.5;           // Scale by 50% of full speed
+    public static final double kSlowClimbScaling      = 0.3;           // Scale by 30% of full speed
```

Comment:
Is this related to Climber? If so, should we include it in the Constants for Swerve?